services/nutanix-ai/metadata.yaml

```diff
@@ -134,12 +135,17 @@ overview: |-
   ### After enabling Nutanix Enterprise AI, Update the endpoint of istio-ingressgateway service external IP or FQDN in the Nutanix AI dashboard ConfigMap.
   On the terminal, run the following command to get the external IP or FQDN of the istio-ingressgateway service from respective cluster:
   ```
-  export NAI_UI_ENDPOINT=https://$(kubectl get svc istio-ingressgateway -n istio-system -ojsonpath='{.status.loadBalancer.ingress[0].ip}')/
+  export NAI_UI_ENDPOINT=https://$(kubectl get svc istio-ingressgateway -n istio-system -o json | jq -r '.status.loadBalancer.ingress[0] | (.hostname // .ip)')/
   ```
   Update the dashboard URL in config map as follows:
   ```
   kubectl patch cm nai-ui -n <workspace_namespace> -p '{"data":{"dashboardLink":"'${NAI_UI_ENDPOINT}'"}}'
   ```
+  Run below patches:
+  ```
+  kubectl patch configmap config-features -n knative-serving --patch '{"data":{"kubernetes.podspec-nodeselector":"enabled"}}'
```

Comment:
About solve the https://jira.nutanix.com/browse/NCN-104322. I think we have discussed it before and I think I should add the override annotation patch so it will not revert by flux. Do you agree on this?

Reply:
okay yeah we can add the override annotation, but we should revert this in long time (and see if switching knative operator will alleviate this ugly workaround)

Reply:
sure. I have checked we need to set this annotation. 
```kustomize.toolkit.fluxcd.io/reconcile: disabled```
But it will disable the flux reconciliation.

Reply:
updated.